src/query/interpret/eval.hpp

```diff
@@ -1165,6 +1170,7 @@ class ExpressionEvaluator : public ExpressionVisitor<TypedValue> {
   FrameChangeCollector *frame_change_collector_;
   /// Property lookup cache ({symbol: {property_id: property_value, ...}, ...})
   mutable std::unordered_map<int32_t, std::map<storage::PropertyId, storage::PropertyValue>> property_lookup_cache_{};
+  bool treat_null_as_zero_;
```

Comment:
`treat_null_as_zero_{false};`

Reply:
actually, it's up to you, since either way it will initialize

Reply:
Consider future code change which adds constructor but forgets to set `treat_null_as_zero_`, passes tests non-deterministically.

IMO where possible, prefer default member initializer.
In the long run it will avoid bugs.